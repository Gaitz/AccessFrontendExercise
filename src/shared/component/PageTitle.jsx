import React from "react";
import styles from "./PageTitle.module.css";

const PageTitle = (props) => {
  return <h2 className={styles.pageTitle}>{props.children}</h2>;
};

export default PageTitle;
