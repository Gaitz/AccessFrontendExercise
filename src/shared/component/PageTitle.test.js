import { render } from "@testing-library/react";
import React from "react";
import PageTitle from "./PageTitle";

describe("PageTitle component test", () => {
  it("snapshot test", () => {
    const { container } = render(
      <PageTitle>
        Test Title <em>sub element</em>
      </PageTitle>
    );
    expect(container).toMatchSnapshot();
  });
});
