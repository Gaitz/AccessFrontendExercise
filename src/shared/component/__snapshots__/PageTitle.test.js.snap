// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageTitle component test snapshot test 1`] = `
<div>
  <h2
    class="pageTitle"
  >
    Test Title 
    <em>
      sub element
    </em>
  </h2>
</div>
`;
